index.d.ts

```diff
@@ -93,7 +93,7 @@ declare const dotProp: {
 	//=> {foo: {bar: {y: 'x'}}}
 	```
 	*/
-	delete(object: {[key: string]: any}, path: string): void;
+	delete(object: {[key: string]: any}, path: string): boolean;
```

Comment:
The doc comment needs to be updated.

Reply:
The typings for `dotProp.set()` don't include `Returns the object.` so to remain consistent, I'm not including the ones for `dotProp.delete()` here. We can either not document it here or change both cases.

Reply:
The difference is that in the "set" case, the type self-documents the return value. Here, it's just a boolean and it's not immediately clear what it means. Then I think we should document both.

Reply:
Done!